bin/registerServerTopology.php

```diff
@@ -188,25 +198,29 @@
  */
 $foundIps = explode(" ", trim(shell_exec("hostname -I")));
 $foundIps = array_combine(range(1, count($foundIps)), array_values($foundIps));
+if (count($foundIps) > 1 && !isset($configOptions['CURRENT_NODE_ADDRESS'])) {
+    $goodIp = false;
 
-$goodIp = false;
+    $ipSelection = 'Found IP on CURRENT NODE:' . PHP_EOL;
+    foreach ($foundIps as $key => $ip) {
+            $ipSelection .= "   [$key]: $ip" . PHP_EOL;
+    }
 
-$ipSelection = 'Found IP on CURRENT NODE:' . PHP_EOL;
-foreach ($foundIps as $key => $ip) {
-        $ipSelection .= "   [$key]: $ip" . PHP_EOL;
-}
+    while (!$goodIp) {
+        echo $ipSelection;
+        $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP ?');
 
-while (!$goodIp) {
-    echo $ipSelection;
-    $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP? ');
 
-    if (!array_key_exists($ipChoice, $foundIps)) {
-        echo 'Bad IP Choice' . PHP_EOL;
-    } else {
-        $goodIp = true;
+        if (!array_key_exists($ipChoice, $foundIps)) {
+            echo 'Bad IP Choice' . PHP_EOL;
+        } else {
+            $goodIp = true;
+        }
     }
+    $serverIp = $foundIps[$ipChoice];
+} else {
+    $serverIp = $foundIps[1];
```

Comment:
not $foundIps[0]. What should contains $foundIps[0] ?

Reply:
nothing, the array is re indexed from 1